grails-bom/build.gradle

```diff
@@ -1,108 +1,203 @@
+
+ext {
+    // When making changes in the dependencyVersions, remember to also update the Grails BOM Documentation:
+    // https://docs.grails.org/snapshot/ref/Dependency%20Versions/Grails%20BOM.html
+    // Grails dependencies are listed in SCOTTL format:
+    // groupId:artifactId-prefixes:artifactId-suffixes:version:versionNameOverride
+    // version name = versionNameOverride ?: artifactId-prefixes[0]
+    dependenciesOverview = """\
+        com.fasterxml.jackson.core:jackson-databind::
+        com.github.ben-manes.caffeine:caffeine::
+        com.github.javaparser:javaparser-core::
+        com.h2database:h2::
+        commons-codec:commons-codec::
+        io.methvin:directory-watcher::
+        jakarta.annotation:jakarta.annotation-api::
+        jakarta.inject:jakarta.inject-api::
+        jakarta.persistence:jakarta.persistence-api::
+        jakarta.servlet:jakarta.servlet-api::
+        jakarta.xml.bind:jakarta.xml.bind-api::
+        jline:jline::
+        net.bytebuddy:byte-buddy::
+        net.java.dev.jna:jna::
+        org.apache.ant:ant:,junit:
+        org.apache.commons:commons-text::
+        org.apache.groovy:groovy:ant,cli-picocli,console,dateutil,json,macro,nio,sql,swing,templates,test,test-junit5,xml:$groovyVersion
+        org.apache.tomcat:tomcat-jdbc:::tomcat
+        org.apache.tomcat.embed:tomcat-embed-core:::tomcat
+        org.apache.tomcat.embed:tomcat-embed-logging-juli::
+        org.asciidoctor:asciidoctorj::
+        org.aspectj:aspectjrt,aspectjweaver::
+        org.fusesource.jansi:jansi::
+        com.github.spotbugs:spotbugs-annotations::
+        org.grails.plugins:async,events:::grails-async
+        org.grails.plugins:converters::
+        org.grails.plugins:gsp::
+        org.grails:grails-async,grails-events:gpars,rxjava,rxjava2::grails-async
+        org.grails:grails-datastore-gorm-hibernate5::
+        org.grails:grails-datastore:async,core,gorm,gorm-async,gorm-support,gorm-rx,gorm-test,gorm-validation,web:$datastoreVersion
+        org.grails:grails-gdoc-engine::
+        org.grails:grails-gradle-plugin::
+        org.grails:grails-testing-support,grails-gorm-testing-support,grails-web-testing-support::
+        org.grails:grails:grails-web-taglib,gsp,web-gsp::gsp
+        org.grails:views-json-testing-support::
+        org.hamcrest:hamcrest-core::$hamcrestVersion
+        org.jsoup:jsoup::
+        org.mongodb:bson:,record-codec:$mongodbJavaDriverVersion
+        org.mongodb:mongodb-driver:core,sync:$mongodbJavaDriverVersion
+        org.objenesis:objenesis::
+        org.slf4j:jcl-over-slf4j::$slf4jVersion
+        org.slf4j:slf4j:api,simple:$slf4jVersion
+        org.spockframework:spock-core,spock-spring:::spock
+        org.springframework.boot:spring-boot-cli:::spring-boot
+        org.springframework:springloaded::
+        org.xhtmlrenderer:flying-saucer-pdf-openpdf::
+        org.yaml:snakeyaml::
```

Comment:
`snakeyaml` is already imported at version `2.2`. We are at `2.3`

Reply:
kept override for now, we can remove later

Reply:
this one prob more important to stay synced. removed.